database/migrations/2023_10_04_172124_add_role_id_to_users_table.php

```diff
@@ -0,0 +1,27 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+return new class extends Migration {
+    /**
+     * Run the migrations.
+     */
+    public function up(): void
+    {
+        Schema::table('users', function (Blueprint $table) {
+            $table->foreignId('role_id')->nullable();
```

Comment:
It cannot be nullable. Add the default of the member role ID.

Also, you are missing a foreign key constraint.

Reply:
Done.